internal/hud/server/logs_reader.go

```diff
@@ -94,17 +90,22 @@ func (ls *LogStreamer) Handle(v proto_webview.View) error {
 
 	return nil
 }
+func StreamLogs(ctx context.Context, host model.WebHost, port model.WebPort, printer *hud.IncrementalPrinter) error {
```

Comment:
model.WebURL might be a better choice here. it will behave correctly if the user specifies 0.0.0.0 (there's a subtle difference between the host we're listening on and the host we're connecting to)

Reply:
ugh it SHOULD be model.WebURL except that encodes the schema too :( i might play with it and split it out into `model.BaseWebURL` and `model.WebURL`

Reply:
oh wait duh I can just manipulate the `url.URL` to change the schema doyyy.